css/styles.css

```diff
@@ -0,0 +1,73 @@
+.body {
+  background-color: #CBFDCB;
+
+}
+.header {
+  background-color: #228B22;
+  padding: 1em;
+  text-align: center;
+}
+.header__nav {
+  font-weight: bold;
+}
+.nav__links {
+  margin: 0 .5em;
+  color: #CBFDCB;
+  font-family: 'Signika', sans-serif;
+}
+.nav__links__current {
+  margin: 0 .5em;
+  color: #FFFFFF;
+  font-family: 'Signika', sans-serif;
+  text-decoration: none;
+}
+.nav__links:hover {
+  margin: 0 .5em;
+  color: #d3d3d3;
+  font-family: 'Signika', sans-serif;
+}
+.main {
+  margin: 0 5%;
```

Comment:
You should consider applying padding/margin with em, rem, or even px. Using percentages may yield unexpected results at some window dimensions.